code/script.js

```diff
@@ -1,213 +1,252 @@
+    eyes: "blue",
+    accessories: ["glasses"],
     other: []
   },
   {
-    name: 'Jocelyn',
-    img: 'images/jocelyn.svg',
-    hair: 'black',
-    eyes: 'brown',
-    accessories: ['glasses'],
+    name: "Jocelyn",
+    img: "images/jocelyn.svg",
+    hair: "black",
+    eyes: "brown",
+    accessories: ["glasses"],
     other: []
   },
   {
-    name: 'Jon',
-    img: 'images/jon.svg',
-    hair: 'brown',
-    eyes: 'green',
-    accessories: ['glasses'],
+    name: "Jon",
+    img: "images/jon.svg",
+    hair: "brown",
+    eyes: "green",
+    accessories: ["glasses"],
     other: []
   },
   {
-    name: 'Jordan',
-    img: 'images/jordan.svg',
-    hair: 'yellow',
-    eyes: 'hidden',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Jordan",
+    img: "images/jordan.svg",
+    hair: "yellow",
+    eyes: "hidden",
+    accessories: ["glasses", "hat"],
+    other: ["active"]
   },
   {
-    name: 'Josephine',
-    img: 'images/josephine.svg',
-    hair: 'grey',
-    eyes: 'brown',
+    name: "Josephine",
+    img: "images/josephine.svg",
+    hair: "grey",
+    eyes: "brown",
     accessories: [],
     other: []
   },
   {
-    name: 'Josh',
-    img: 'images/josh.svg',
-    hair: 'yellow',
-    eyes: 'green',
+    name: "Josh",
+    img: "images/josh.svg",
+    hair: "yellow",
+    eyes: "green",
     accessories: [],
     other: []
   },
   {
-    name: 'Jude',
-    img: 'images/jude.svg',
-    hair: 'black',
-    eyes: 'green',
+    name: "Jude",
+    img: "images/jude.svg",
+    hair: "black",
+    eyes: "green",
     accessories: [],
-    other: []
+    other: ["non-smoker"]
   },
   {
-    name: 'Julie',
-    img: 'images/julie.svg',
-    hair: 'black',
-    eyes: 'brown',
-    accessories: ['glasses', 'hat'],
-    other: []
+    name: "Julie",
+    img: "images/julie.svg",
+    hair: "black",
+    eyes: "brown",
+    accessories: ["glasses", "hat"],
+    other: ["active"]
   },
-]
+];
+
+// Ylva - here are all the Global variables declared
+let secret; //this is the secret person
+let currentQuestion; //current question object
+let charactersInPlay; //the array of characters in the game
+let numberOfQuestions; //number of questions allowed
+let timeLeft; //time left in the timer
+let timerCountdown; //countdown timer
+let playerName; //the user's name in the game
 
-// Global variables
-let secret
-let currentQuestion
-let charactersInPlay
+// Ylva - might add sound effects here if I have time later
+
+// Ylva - Function for the timer function (to show double digits)
+const paddedNumber = (number, length) => {
+  let str = number + "";
+  while (str.length < length) str = "0" + str;
+  return str;
+};
+// countdown- function and reset for the timer
+const resetTimer = () => {
+  timeLeft = 240;
+  //timeLeft = 120; //2-minutes countdown for game - might make this longer
+
+  countDown = setInterval(() => {
+    if (timeLeft <= 0) {
+    clearInterval(countDown);
+    winOrLoseText.innerHTML = `<p>Game Over</p>`;
```

Comment:
Since your game has two ways to get a game over (guessing the wrong person and having the timer run out), a good idea would be to make it a separate function so you won't have to have duplicate code.

Reply:
@edv-rd Great idea!
Do you know how I can make that?
Any suggestions would be appreciated, I haven't got a clue actually on how to write that kind of function for two different scenarios with the same outcome just now but ofcourse I want to keep the code clean and avoid to much repitition...